src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -31,6 +33,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
 
     String username = request.getParameter("username");
     String password = request.getParameter("password");
+    String about = request.getParameter("about");
+    boolean isAdmin = false;
+
+    if (request.getParameter("admin_password") == ADMIN_PASSWORD) {
```

Comment:
ADMIN_PASSWORDS.equals(request.getParameter("admin_password"))

- Avoid string equality with ==, use equals
- Always use the constant firs to avoid NPEs